railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
@yuki24 but context switching is going to be triggered either way. So, if you are I/O bound (and can afford it) you may have more processes than cores in the same proportion you can have more threads than cores.

My understanding is that number of cores is relevant in both models, and also mixed models (*N* processes with *M* threads), because in the end threads is all a CPU runs.